### PR TITLE
make travis only run linters/auto-gen checkers.

### PR DESCRIPTION
**Pull request by lavalamp**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,39 +5,21 @@ language: go
...
-  - go get github.com/mattn/goveralls
   - go get github.com/tools/godep
   - ./hack/travis/install-etcd.sh
+  - ./hack/build-go.sh
+
+script:
   - ./hack/verify-gofmt.sh
   - ./hack/verify-boilerplate.sh
   - ./hack/verify-description.sh
   - ./hack/travis/install-std-race.sh
```
Comment (thockin) [inline .travis.yml]: do we need race?

File: .travis.yml [hunk 2]
```diff
@@ -5,39 +5,21 @@ language: go
...
-          KUBE_TEST_ETCD_PREFIXES="kubernetes.io/registry"
 
 install:
-  - go get golang.org/x/tools/cmd/cover
-  - go get github.com/mattn/goveralls
   - go get github.com/tools/godep
   - ./hack/travis/install-etcd.sh
+  - ./hack/build-go.sh
+
+script:
```
Comment (thockin) [inline .travis.yml]: This is strictly better than mine because mine has an implied "make" which is redundant.

File: .travis.yml (hunk 1 above)
Comment (lavalamp) [inline .travis.yml]: oh, probably not

Comment (lavalamp) [inline .travis.yml]: Removed, let's see if it still works.

**Comment by thockin** [comment]
LGTM, assuming it passes it is OK to merge

**Comment by lavalamp** [comment]
Second try ran in 3 minutes. Not too bad. Could potentially get it down further by not building every binary.

**Comment by lavalamp** [comment]
@erictune Can you merge this soon so travis doesn't get impossibly backed up now that we've turned it back on?

**Pull request merged**